crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -4141,6 +4141,8 @@ spec:
                   of a devworkspace template.
                 properties:
                   attributes:
+                    additionalProperties:
```

Comment:
TODO investigate `x-kubernetes-preserve-unknown-fields` as `additionalProperties.x-kubernetes-preserve-unknown-fields`. Do we need both?

Reply:
Well, I tried to avoid having this but seems it's OK and related to the latest operator-sdk version, like for DWR we have it for a long time https://github.com/devfile/devworkspace-operator/pull/303